components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -844,17 +844,6 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         await this.userDeletionService.deleteUser(user.id);
     }
 
-    private async getTeamMembersByProject(projectId: string | undefined): Promise<TeamMemberInfo[]> {
```

Comment:
this was unnecessariuly going through project. We now have an organizationId on workspaces so I replaced these with `organizationService.listMembers`.